content/2022/videookulografija.md

```diff
@@ -1,4 +1,561 @@
+
+
+Optimalni vektor parametara $b$ dobija se iz formule za pseudo inverziju koja glasi: 
+
+
+$b = (X^T \cdot X)^{-1} \cdot Y \cdot X^{T}$ 
+
+
+Gde je X matrica koordinata zence oka, a Y matrica koordinata tačaka na ekranu u koje je predviđeno da se gleda tokom prethodno pomentuog eksperimenta.
+
+
+Ove matrice se dobijaju iz baze podataka generisane tokom prethodno pomenutog eksperimenta.
+Dobijen vektor koejficijenta modela b se koristi za izračunavanje koordinta tačaka na ekranu u koje je zenica uperena.
+Ovako dobijen vektor koeficijenta b nije deltvoran.
+Konveksni oblik oka izaziva nelinarno kretanje zenice i ugao između ravni pogleda i ravni ekrana nije normalan zbog čega je potrebno koristiti polinomialnu regresiju.
+
+
+#### 3.2.3.2. Polinomialna regresija
+
+
+Sistem nije linearan zbog konveksnog oblika oka i ugla između ravni pogleda i ravni ekrana koji nije normalan zbog čega se koristi polinomialna regresija.
+Poliomilna regresija je specijalan slučaj višestruke linearne regresije gde se modeluje veza između zavisne i nezavisne promenljive, u ovom slučaju, između koordinata zenice oka i koordinate pogleda na ekranu, kao polinom n-tog stepena. 
+U ovom slučaju, nema linearne korelacije između promenljivih (slika 19.1), pa se radi veće preciznosti primenjuje poliomilna regresija (slika 19.2).
+
+
+![Primer linearne i polinomialne regresije - slika 19](/images/2022/videookulografija/polinomialnaRegresija.png)(slika 19)
+
+
+Za razliku od linearne regresije, kod polinomialne regresije se nezavisna promenljiva x stepenuje stepenom regresije n i zbog toga se dobija bolja korelacija između promenjljivih.
+
+
+Formula polinomialne regresije glasi:
+
+
+$y= b_0+b_1 \cdot x_1+ b_2 \cdot x_1^2 + b_3  \cdot x_1^3+ ... + b_n \cdot x_1^n$
+
+gde su:
+
+
+b - koeficijenti regresije
+
+
+n - stepen regresije (u slučaju ovog rada $n = 3$)
+
+Kako bi izračunali optimalan koeficijent regresije neophodno je da se polinomialna regresija svede na lineanrnu i nakon toga korišćenjem pseudo inverzije izračuna optimalan koeficijent b.
+Polinomialnu regresiju se svodi na linearnu regresiju tako što se konstruiše matricu X: 
+
+$\begin{bmatrix}
+   1 & x_1 & x_1^2 & ... & x_1^n \\
+   1 & x_2 & x_2^2 & ... & x_2^n \\
+   1 & x_3 & x_3^2 & ... & x_3^n \\
+   . & . & . & . & . \\
+   . & . & . &  .  & . \\
+   . & . & . &   . & . \\
+   1 & x_m & x_m^2 & ... & x_m^n 
+\end{bmatrix}$
+
+U ovoj matrici n predstavlja stepen polinomialne regresije.
+Ovakvu matricu ćemo iskoristiti u formuli za pseudo inverziju i na taj način dobiti optimalni koeficijent regresije $b$.
+
+Formula za pseudo inverziju:  $b = (X^T \cdot X)^{-1} \cdot Y \cdot X^{T}$ 
+
+
+## **4. Rezultati i istraživanja**
+
+
+Model dobijen poliomilnom regresijom je proveren tako što je eksperiment snimiljen još jednom, ali ovog puta je dobijen predviđen položaja zenice.
```

Comment:
ajde malo lepse ovo. tipa model je verifikovan ponovnim snimanjem (ili tako nešto)